draft-packages/collapsible/KaizenDraft/ExpertAdviceCollapsible/ExpertAdviceCollapsible.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import { render } from "@testing-library/react"
+import { configure, queryByTestId } from "@testing-library/dom"
+import * as React from "react"
+import { ExpertAdviceCollapsible } from ".."
+
+configure({ testIdAttribute: "data-automation-id" })
```

Comment:
Generally in Kaizen we avoid using test id unless entirely necessary. There are some older tests that still do this, but where possible we use getByRole or getByText to align close to test what a user would be looking for.

Reply:
I've updated these but would love more feedback if it can be improved further 🙏